circe-scalafx/src/main/scala/circefx/Circefx.scala

```diff
@@ -0,0 +1,27 @@
+package circefx
+
+import fx.*
+import fx.Control
+import fx.Control.*
+import io.circe.*
+import cats.*
+import cats.data.*
+import cats.implicits.*
+
+import Continuation.*
+
+object Circefx:
+  def decodeTest[A](c: io.circe.HCursor)(using d: Decoder[A], cx: Control[DecodingFailure]): A =
+    d(c).bind
+
+  def parsing[A](s: String)(using p: Parser, cx: Control[Error], d: Decoder[A]): A =
+    p.decode(s).bind
+
+  object extensionTest:
+    def foldNonEmpty[E, A](x: NonEmptyList[E])(using control: Control[NonEmptyList[E]]): A =
+      // summon[Control[NonEmptyList[E]]]
+      ???
+
```

Comment:
@jackcviers I noted on using [fold](https://www.javadoc.io/static/org.typelevel/cats-docs_2.13/2.8.0/cats/data/Validated.html#fold%5BB%5D(fe:E=%3EB,fa:A=%3EB):B)
now that leaves me with a sort of cast I need to do from `NonEmptyList[E]` to `A` as expressed in the `foldNonEmpty`. I am sure here I need to use the `shift`.